runtimes/pythonrt/runner/tee.py

```diff
@@ -0,0 +1,51 @@
+from queue import Queue
+from threading import Thread
+from time import sleep
+import grpc
+import pb
+
+
+class Tee:
```

Comment:
Add docstring on what it does.

Reply:
Also unit tests? This is a general-purpose class that doesn't have any AK complexities that make testing harder.